### ISSUE TITLE
Invalid link can impact valid link

### ISSUE THREAD

**Issue by BlackWolf** [jsdoc3/jsdoc#936]
Consider the following example:

```
/**
 * @class
 * @classdesc Link to {@link Test#method2} and {@link Test#method}
 */
function Test() {
}

/**
 * Documented method
 */
Test.prototype.method = function() {
};

Test.prototype.method2 = function() {
};
```

In this scenario, the second link will appear as the jsdoc command (`{@link Test#method`}) in the output, even though it is valid. This behaviour seems to be caused by the first link not being valid. If `method2` is also documented, both links appear correctly in the documentation.

Although, of course, a good documentation documents all methods, this still seems like an undesirable bug that could lead to confusion.

**Comment by hegemonic** [comment]
Confirmed. This looks like an edge case that isn't covered by our current test suite.

**Comment by hegemonic** [comment]
Fixed on master and releases/3.3. The fix will be included in JSDoc 3.3.0.
